services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -367,9 +367,9 @@ getValidateSAMLEmailsInternal ::
   ( Member TeamFeatureStore r,
     Member (Input Opts) r
   ) =>
-  FeatureGetter 'WithoutLockStatus 'TeamFeatureValidateSAMLEmails r
+  FeatureGetter 'WithLockStatus 'TeamFeatureValidateSAMLEmails r
 getValidateSAMLEmailsInternal =
-  Tagged $ getFeatureStatusWithDefaultConfig @'TeamFeatureValidateSAMLEmails flagsTeamFeatureValidateSAMLEmailsStatus . mbTeam
+  Tagged $ getFeatureStatusWithDefaultConfigAndImplicitLockStatus @'TeamFeatureValidateSAMLEmails flagsTeamFeatureValidateSAMLEmailsStatus . mbTeam
```

Comment:
I don't understand.  I thought the point was to get rid of the concern where the lock status is coming from, and always have one at this point.  Can we somehow get rid of `getFeatureStatusWithDefaultConfig`?  If so, I would use that name, but I don't feel strongly about it.  Long names are helpful!  :)

Reply:
The question is if we want to update the DB schema with the lock status for all feature even when we do not want to support changing it. If that is the case, then I agree, otherwise we need to handle the case separately.